client/src/GameState.ts

```diff
@@ -121,6 +158,7 @@ export class GameState {
             this.currentTetromino.reportState()
         );
         this.placeTetromino(this.currentTetromino);
+        this.clearTradeAndEmit();
```

Comment:
Will this clear the trades for everyone? i.e. If player 1 puts their piece up to trade but player 2 places, won't that erroneously cancel player 1's trade?